ntcore/src/main/java/edu/wpi/first/networktables/NetworkTable.java

```diff
@@ -269,9 +297,64 @@ public boolean containsKey(String key) {
    *     its own
    */
   public boolean containsSubTable(String key) {
-    int[] handles =
-        NetworkTablesJNI.getEntries(m_inst.getHandle(), m_pathWithSep + key + PATH_SEPARATOR, 0);
-    return handles.length != 0;
+    Topic[] topics = m_inst.getTopics(m_pathWithSep + key + PATH_SEPARATOR, 0);
+    return topics.length != 0;
+  }
+
+  /**
+   * Gets topic information for all keys in the table (not including sub-tables).
+   *
+   * @param types bitmask of types; 0 is treated as a "don't care".
```

Comment:
List of supported flags?

Reply:
I can add a link to NetworkTableType, because that's what this is.